buildbot/osuosl/master/config/builders.py

```diff
@@ -572,15 +573,21 @@
     'builddir': "clang-arm64-windows-msvc-2stage",
     'factory' : ClangBuilder.getClangCMakeBuildFactory(
                     vs="manual",
-                    clean=False,
+                    clean=True,
                     useTwoStage=True,
-                    checkout_flang=True,
                     testStage1=False,
                     extra_cmake_args=[
+                        "-DLLVM_TARGETS_TO_BUILD=AArch64",
                         "-DCLANG_DEFAULT_LINKER=lld",
                         "-DCMAKE_TRY_COMPILE_CONFIGURATION=Release",
                         "-DCMAKE_C_COMPILER_LAUNCHER=ccache",
                         "-DCMAKE_CXX_COMPILER_LAUNCHER=ccache",
+                        "-DCOMPILER_RT_BUILD_SANITIZERS=OFF"],
+                    extra_stage2_cmake_args=[
+                        "-DLLVM_TARGETS_TO_BUILD='ARM;AArch64;X86'",
+                        "-DLLVM_ENABLE_PROJECTS=flang;mlir",
```

Comment:
Is this additive or replacing the projects?

I assume we should have clang and maybe lld in this list.

Reply:
I think it will replace `-DLLVM_ENABLE_PROJECTS` arg to whatever is defined in `f.enable_projects` (https://github.com/llvm/llvm-zorg/blob/9db7725faca7f025d788286d230f902e479c6c35/zorg/buildbot/builders/ClangBuilder.py#L438). But even if it would work as you think it should, then you build projects commits to which do not trigger the build. leading to false blame lists in case of broken flang or mlir.